### ISSUE TITLE
Specifying multiple items on the --pythonwarnings pytest option in the makefile

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#1720]
The current makefile sets the pytest option for warnings as follows for py3 and py27:

```
--pythonwarnings=default,ignore::DeprecationWarning,ignore::PendingDeprecationWarning
```

This causes the following issue (at least on Python 2.7.5 as reported in issue #1714, and verified on Python 2.7.16 on MacOS):

```
py.test --color=yes --pythonwarnings=default,ignore::DeprecationWarning,ignore::PendingDeprecationWarning -v --tb=short tests/end2endtest -s
============================================================================ test session starts =============================================================================
platform linux2 -- Python 2.7.5, pytest-4.4.0, py-1.8.0, pluggy-0.9.0 -- /usr/bin/python2
cachedir: .pytest_cache
rootdir: /root/pywbem
plugins: cov-2.5.1
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR> File "/usr/lib/python2.7/site-packages/_pytest/main.py", line 209, in wrap_session
INTERNALERROR> session.exitstatus = doit(config, session) or 0
INTERNALERROR> File "/usr/lib/python2.7/site-packages/_pytest/main.py", line 248, in _main
INTERNALERROR> config.hook.pytest_collection(session=session)
INTERNALERROR> File "/usr/lib/python2.7/site-packages/pluggy/hooks.py", line 289, in call
INTERNALERROR> return self._hookexec(self, self.get_hookimpls(), kwargs)
INTERNALERROR> File "/usr/lib/python2.7/site-packages/pluggy/manager.py", line 68, in _hookexec
INTERNALERROR> return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR> File "/usr/lib/python2.7/site-packages/pluggy/manager.py", line 62, in
INTERNALERROR> firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
INTERNALERROR> File "/usr/lib/python2.7/site-packages/pluggy/callers.py", line 208, in _multicall
INTERNALERROR> return outcome.get_result()
INTERNALERROR> File "/usr/lib/python2.7/site-packages/pluggy/callers.py", line 81, in get_result
INTERNALERROR> _reraise(*ex) # noqa
INTERNALERROR> File "/usr/lib/python2.7/site-packages/pluggy/callers.py", line 182, in _multicall
INTERNALERROR> next(gen) # first yield
INTERNALERROR> File "/usr/lib/python2.7/site-packages/_pytest/warnings.py", line 150, in pytest_collection
INTERNALERROR> config=config, ihook=config.hook, when="collect", item=None
INTERNALERROR> File "/usr/lib64/python2.7/contextlib.py", line 17, in enter
INTERNALERROR> return self.gen.next()
INTERNALERROR> File "/usr/lib/python2.7/site-packages/_pytest/warnings.py", line 90, in catch_warnings_for_item
INTERNALERROR> warnings._setoption(arg)
INTERNALERROR> File "/usr/lib64/python2.7/warnings.py", line 120, in _setoption
INTERNALERROR> action = _getaction(action)
INTERNALERROR> File "/usr/lib64/python2.7/warnings.py", line 145, in _getaction
INTERNALERROR> raise _OptionError("invalid action: %r" % (action,))
INTERNALERROR> _OptionError: invalid action: 'default,ignore'

======================================================================== no tests ran in 0.11 seconds ========================================================================
```

Setting PYTHONWARNINGS to that same value works, so this is an issue with pytest not supporting comma separated lists.

Using the bash syntax for setting the env var (i.e. `PYTHONWARNINGS=... py.test ...`) is not possible because the makefile is used also on native Windows.

PR #1719 mitigates the issue by temporarily reducing the items to just one.

Need to figure out what to do about this.

**Comment by andy-maier** [comment]
Reading https://docs.python.org/3/using/cmdline.html#envvar-PYTHONWARNINGS, it seems that the syntax with a single string with comma-separated specifiers that can be used in the env var needs to split up into multiple option uses when using the -W option. Need to try that out with pytest.

**Comment by andy-maier** [comment]
This was fixed by using single `-W` options for each setting. Did not enable all earlier suppressions, but only suppression of ResourceWarning on py3 and PendingDeprecationWarning on all versions. DeprecationWarning is not suppressed anymore because meanwhile the messages announce in which Python version the deprecated feature goes away.

PR #1868 fixes this for master, and PR #1867 for 0.14.6.
